src/server/api/controllers/answers.controller.js

```diff
@@ -0,0 +1,34 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getAnswers = async () => {
+  return knex('answers');
+};
+
+const getAnswerById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const answer = await knex('answers')
+      .select('answers.id as id', 'answer')
+      .where({ id });
+    if (answer.length === 0) {
+      throw new Error(`incorrect entry with the id of ${id}`, 404);
+    }
+    return answer;
```

Comment:
This still returns an array is that what you would expect using the endpoint? :)

Reply:
Thanks Niels, I made the requested changes to return just an object instead of an array :)